scot/datasets.py

```diff
@@ -0,0 +1,85 @@
+# Released under The MIT License (MIT)
+# http://opensource.org/licenses/MIT
+# Copyright (c) 2016 SCoT Development Team
+
+from os import makedirs
+from os.path import expanduser, isfile, isdir, join
+from requests import get
+import numpy as np
+
+from .matfiles import loadmat
+from .eegtopo.eegpos3d import positions
+from . import config
+
+
+datadir = expanduser(config.get("scot", "data"))
+valid_datasets = ["mi"]
+files = {"mi": ["motorimagery.mat"]}
+url = {"mi": "https://github.com/scot-dev/scot-data/raw/master/scotdata/"}
```

Comment:
Instead of multiple dicts I would prefer something like `datasets = {"mi": {"files": ..., "url": ...}}`.

Both approaches have their benefits. I'd just prefer not having to edit multiple placed when adding another dataset in the future.